src/NovelBasicFillRect.cpp

```diff
@@ -20,22 +19,30 @@ void NovelBasicFillRect::drawObject() {
     return;
 
   glUseProgram(_programId);
+
+/*  glBindBuffer(GL_UNIFORM_BUFFER, _modelTransformUniformBuffer.getActual());
+  GLvoid* p = glMapBuffer(GL_UNIFORM_BUFFER, GL_WRITE_ONLY);
+  memcpy(p, &_modelTransform.getActual(), sizeof(_modelTransform.getActual()));
+  glUnmapBuffer(GL_UNIFORM_BUFFER);*/
+
+  glUniformMatrix3fv(1, 1, GL_FALSE, &_modelTransform.getActual()[0][0]);
```

Comment:
Since `_modelTransform` is just a `mat3`, you should be able to take the address of it directly.

Reply:
This code will not exist once the UBO is fully implemented.